Podcast/FetchUserFollowersByIDRequest.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FetchUserFollowersByIDRequest.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 9/13/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+enum UserFollowsType {
+    case Followers
+    case Followings
+}
+
+class FetchUserFollowsByIDRequest: EndpointRequest {
+    
+    var userId: String
+    var type: UserFollowsType
+    
+    init(userId: String, type: UserFollowsType) {
+        
+        self.userId = userId
+        self.type = type
+        super.init()
+        
+        path = type == .Followers ? "/followers/show/\(userId)/" : "/followings/show/\(userId)/"
+        //"followings/show?id=id"
+        httpMethod = .get
+        //queryParameters = ["id": userId]
+    }
+    
+    override func processResponseJSON(_ json: JSON) {
+        //don't really need these b/c same user,session returned
+        let key = type == .Followers ? "followers" : "followings"
+        let userKey = type == .Followers ? "follower" : "followed"
```

Comment:
I'm not sure what you mean? I need these string variables?

Reply:
I mean in our enum UserFollowsType we can create var key on the enum that returns a certain value for each type (https://stackoverflow.com/questions/28393904/enum-variables-in-swift) .. instead of containing the logic for an enum in the processResponseJOSN function